components/toolbar.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { StyleSheet, Text, View, FlatList, ScrollView, Image } from 'react-native';
+
+export default class App extends React.Component {
+  render() {
+    return (
+        <View> 
+            <View style={styles.viewToolbarBackground} >
+                <View style = {styles.viewToolbarCart}> </View>
+                <View style = {styles.viewToolbarHome}> </View>
+                <View style = {styles.viewToolbarPerson}> </View>
+            </View>
+        </View> 
+          );
+        }
+      }
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        backgroundColor: '#000',
+    },
+    viewToolbarBackground: {
```

Comment:
Make sure that you are tabbing the content of these style objects. Lines 20 and 21 are properly indented.